lean/main/03_expressions.lean

```diff
@@ -285,8 +285,8 @@ def addOne : Expr :=
     BinderInfo.default
 
 def mapAddOneNil : Expr :=
-  mkAppN (.const ``List.map [levelOne, levelOne])
-    #[nat, nat, addOne, .app (.const ``List.nil [levelOne]) nat]
+  mkAppN (.const ``List.map [levelZero, levelZero])
+    #[nat, nat, addOne, .app (.const ``List.nil [levelZero]) nat]
```

Comment:
This makes the universe output differ from what's shown below a few lines down on line 304:

`-- @List.map.{1, 1} Nat Nat (fun x => Nat.add x 1) (@List.nil.{1} Nat) : List.{1} Nat`

I fully defer to the Zulip thread that this change is right though -- can you just update that line so its output now has the right universes in it as well, and then this should be good to merge.

Reply:
Sorry about missing this and great you noticed! Updated.